docs/rules/no-useless-escape.md

```diff
@@ -26,7 +26,8 @@ Examples of **incorrect** code for this rule:
 `\#{foo}`;
 /\!/;
 /\@/;
-
+/[\[\]]/;
```

Comment:
This example might be misleading because only one of the two escapes is useless, so it might be better to remove the other from the example?

```suggestion
/[\[]/;
```

Reply:
Should the “correct” example also be adjusted? When the “incorrect” and “correct” examples are compared, it’s clear which escape is useless. I was kind of trying to include realistic examples and I’m not sure if `/[\[]/` qualifies. Still, not sure what the best set of examples is.

Reply:
We can add to “correct” examples both `/[[]/;` to mirror the incorrect `/[\[]/;`, and `/[[\]]/;` to show that `\]` isn't useless.

Reply:
It looks a bit redundant this way. I think it looks good if in “incorrect”, there’s the example `/[\[]/;`, and in “correct”, there’s the two examples `/[[]/;` and `/[\]]/`.

(By the way, sorry for the delay. This week was busy…)